### PR TITLE
Add tagging feature to frontend

### PR DESCRIPTION
**Pull request by yigitozkavci**
This PR implements tagging feature on front-end.

image

### CODE REVIEW

File: Atlas/frontend/src/containers/CulturalHeritage/Add.js [hunk 1]
```diff
@@ -73,7 +68,8 @@ const mapDispatchToProps = dispatch => {
         headers: { 'Authorization': 'JWT ' + token },
         data: {
           title: addCHInputs.title,
-          description: addCHInputs.description
+          description: addCHInputs.description,
+          tags: addCHInputs.tags.map(t => ( { name: t.text } ))
```
Comment (YmirKhang) [inline Atlas/frontend/src/containers/CulturalHeritage/Add.js:72]: image
I am getting an error here

File: Atlas/frontend/src/components/CulturalHeritage/Add.js [hunk 2]
```diff
@@ -41,6 +42,16 @@ const Page = ({ user, token, addCHInputs, addCHErrors, handleCHInputChange, addC
               </Col>
           </Row>
       </FormGroup>
+      <FormGroup>
+          <Row>
+              <Col xs="3">
+                  <label>Tags:</label>
+              </Col>
+              <Col xs="9">
+                <ReactTags tags={ addCHInputs.tags || [] } handleAddition={ addCHTag } handleDelete={ deleteCHTag }/>
```
Comment (YmirKhang) [inline Atlas/frontend/src/components/CulturalHeritage/Add.js]: Should there be a button for adding tags.

**Review by YmirKhang — CHANGES_REQUESTED** [review]
Getting errors while adding

File: Atlas/frontend/src/containers/CulturalHeritage/Add.js (hunk 1 above)
Comment (yigitozkavci) [reply Atlas/frontend/src/containers/CulturalHeritage/Add.js:72]: Can you show the error message? It's in the upper part of your screen shot, and missing :/

**Review by YmirKhang — DISMISSED** [review]
Okay working

**Comment by yigitozkavci** [comment]
I've added a few improvements. Can you review again @emir350z ?

**Review by YmirKhang — APPROVED** [review]

**Pull request merged**